CHANGELOG.md

```diff
@@ -0,0 +1,114 @@
+# Changelog
```

Comment:
there already is a CHANGES.md file i n the repo, it should be taken into account before merging this PR (rename changes to CHANGELOG?)